src/preferences.html

```diff
@@ -2,7 +2,10 @@
 <html data-theme="" lang="en">
 
 <head>
+    <meta charset="utf-8">
     <meta http-equiv="Content-Security-Policy" content="script-src 'self'">
+    <meta name="description" content="Time to Leave, the app that tells you when to go home">
```

Comment:
```suggestion
    <meta name="description" content="Time to Leave, the app that logs your working hours and notifies you when it's time to leave the office and start to live">
```

I think we should use our "tagline" here. Do we need descriptions on all windows though?

Reply:
As far as standards go, yes we need descriptions on all windows. Realistically, no not really. And definitely the tagline is a good call. I'm not actually sure in what situation screen readers need the description.